lib/stoplight/domain/strategies/red_run_strategy.rb

```diff
@@ -10,6 +10,14 @@ module Strategies
       #
       # @api private
       class RedRunStrategy < RunStrategy
+        # @!attribute [r] config
+        #   @return [Stoplight::Domain::Config] The configuration for the light.
+        protected attr_reader :config
+
+        def initialize(config:)
+          @config = config
```

Comment:
Do we want to add super here and in other strategies to make strategies more extensible in the future?

Reply:
Actually, I removed the `RunStrategy#initialize` method because different strategies have different dependencies. We could reconsider his if a common initialization logic arises